includes/Story_Post_Type.php

```diff
@@ -398,6 +398,15 @@ public static function get_editor_settings() {
 					'fallbackPoster'  => plugins_url( 'assets/images/fallback-poster.jpg', WEBSTORIES_PLUGIN_FILE ),
 				],
 			],
+			'flags'  => [
```

Comment:
At some point, we could have a UI for this in WP, we can enable / disable flags and it persists.

Reply:
Yep, that's what I mentioned in my comment too 👍